docs/tutorials/02_installation.md

```diff
@@ -0,0 +1,112 @@
+# 2. Installation
+
+Setup for both x86-based and Jetson-based ECUs.
+
+## 2-1. x86-based ECU
+
+**NOTE: Network settings are automatically updated**
+
+During this procedure, IP addresses are assigned to some network interfaces (refer to the connection diagram on [1.Hardware setup](01_hardware_setup.md) for more detail) using `netplan`.
+This behavior may cause unexpected disconnection, if you are accessing the ECU remotely via those interfaces.
+
+If you would like to change network interfaces or IP addresses to be assigned, edit `edge-auto/ansible/playbooks/vars/edge_auto.yaml` before running `setup-dev-env.sh`
+
+### Download the repository and setup your environment
+
+As a first step, clone `tier4/edge-auto` and move to the directory.
+
+```sh
+git clone https://github.com/tier4/edge-auto.git
+cd edge-auto
+```
+
+You can install the dependencies using the provided ansible script.
+
+```sh
+./setup-dev-env.sh
+```
+
+Finally, please reboot the system to make the installed dependencies and permission settings effective.
+
+```sh
+sudo reboot
+```
+
+### Build edge-auto
+
+Create your ROS workspace and clone repositories using vcstool.
+
+```sh
+cd edge-auto
+mkdir src
+vcs import src < autoware.repos
+```
+
+Install ros package dependencies and build your ROS workspace.
+
+```sh
+rosdep install -y -r --from-paths src --ignore-src --rosdistro $ROS_DISTRO
+
+colcon build \
+  --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release \
+  --packages-up-to edge_auto_launch
+```
+
+## 2-2. Jetson-based ECU
+
+**This following steps can be performed from your x86-based ECU via ssh.**
```

Comment:
```suggestion
**This following steps can be performed from your x86-based ECU via ssh. Besides, these steps require internet connection**
```
@aohsato As you modified, this setup procedure can be performed via ssh. In addition to that, I would like to let users know that this procedure requires an internet connection to download packages (I especially would like to mention that because the Jetson-based ECU in the system overview is only connected to x86-based ECU)

Reply:
@manato Correct. How about adding ethernet cable for internet connection on the system figure in `1.Hardware step`? It just for installation step.

Reply:
added [84f60da](https://github.com/tier4/edge-auto/pull/3/commits/84f60da42bf2ab63b9536f220093fd8c233e21bd)